### ISSUE TITLE
O> Use GitToken for features in Decentralised Accounting and Spending

### ISSUE THREAD

**Issue by lapin7** [rchain/Members#101]
https://gittoken.org/

[These things](https://github.com/git-token/documentation/wiki/Token-Issuance-and-Distribution-Model) are for example monitored and could quantify the level of participation on an issue.

**Comment by Ryanmtate** [comment]
Hi All,

I am looking forward to setting up RChain's contribution accounting process with GitToken. 

@lapin7 and I spent some time today discussing RChain's current proposal process and ideating around using GitToken for tracking and quantifying GitHub contribution activities. Would love to use the RChain project as a means to further identify features of value for development that may be helpful for other projects.

I am about two weeks away from an alpha release. I am currently in the middle of a refactor to make onboarding GitHub organizations easier; e.g. provide hosting to organizations that may not want to run a private service.

Here's a current list of things I am working towards integrating by September 30th

- [  ] Publish git-token @alpha tag; Current version can be installed using npm or yarn: `npm i -g git-token@0.0.37`

- [X] CLI Registration => `git token register`

- [X] Hosted Webhook service => Point webhook service at `https://webhook.gittoken.io/:organization`

- [  ] Terminal dashboard (WIP) => `git token terminal` will bring up a terminal dashboard for viewing and analyzing GitToken projects. CLI is currently not set up to include

**Comment by Ryanmtate** [comment]
Here is an example registration process. 

Note: Must be an admin to register the organization with GitToken.

![git-token-register](https://user-images.githubusercontent.com/2837196/30498735-a9197010-9a0c-11e7-8a76-306e09da8802.gif)

**Comment by dckc** [comment]
Noone has touched this in months. Is it time to close as wontfix?

I'm not inclined to tie rewards so closely to mechanical actions such as commits, comments, etc. A lot of our issues are actually placeholders for work that mostly happens elsewhere.

**Comment by Ryanmtate** [comment]
GitToken is still being developed; ultimately not worth waiting on using if there is work that needs to be accounted for now.

**Comment by dckc** [comment]
We can come back to this later if we like.
